src/models/Call.ts

```diff
@@ -899,13 +900,15 @@ export class ElementCall extends Call {
                 MatrixRTCSessionEvent.MembershipsChanged,
                 (_, newMemberships: CallMembership[]) =>
                     newMemberships.some((m) => m.sender === this.client.getUserId()),
+                60000, // allow user to wait in the lobby for 1 min
```

Comment:
Users can and will wait on this screen for longer than 1 minute (especially in video rooms), so can we remove the timeout?

Reply:
Updated it to also allow using waitForEvent without timeout.